lib/phlex/sgml.rb

```diff
@@ -566,14 +566,20 @@ def __nested_attributes__(attributes, base_name, buffer = +"")
 		attributes.each do |k, v|
 			next unless v
 
-			name = case k
-				when String then k
-				when Symbol then k.name.tr("_", "-")
-				else raise Phlex::ArgumentError.new("Attribute keys should be Strings or Symbols")
-			end
+			if k == :_
```

Comment:
Given that we have to check this twice, it may be faster to store the result in a local
```suggestion
			if root_key = (k == :_)
```

Reply:
It may also be better to flip this equality check around since we always know what `:_` is.
```suggestion
			if :_ == k
```